konan_sdk/endpoints/base_endpoint.py

```diff
@@ -165,8 +165,33 @@ def headers(self) -> Dict:
         }
 
 
-class KonanBaseDeploymentEndpoint(KonanBaseAuthenticatedEndpoint[ReqT, ResT]):
-    """Base Endpoint class for Konan endpoints that deal with deployments.
+class KonanBaseGenericDeploymentsEndpoint(
+    KonanBaseAuthenticatedEndpoint[ReqT, ResT]
```

Comment:
I feel that adding that class just for the endpoint path is good enough, since it will ensure that any subsequent deployment-related endpoints that don't expect a `deployment_uuid` argument will have a convenient parent class to inherit from.

For now, only creating a new deployment endpoint, and its corresponding class `CreateDeploymentEndpoint` is benefiting from this design, but surely it will come handy in the future when extra such endpoints are added.